### DISCUSSION TITLE
Sign in failed: Error: network error. 2024-05-24 10:05:25.014 [error] [auth] Extension activation failed: "connect ETIMEDOUT 3.127.156.149:443"

### DISCUSSION THREAD

**Discussion by michaeltaman** [community/community#125527] [Copilot Conversations] [answered]
![image](https://github.com/community/community/assets/42732094/fa43a8e0-9bbf-4f43-a29c-64a53a4ed1c2)

**I haven't been able to log in for 2 days now !!!**

My VSCode version is latest:
---------------------------------
Version: 1.89.1 (user setup)
Commit: dc96b837cf6bb4af9cd736aa3af08cf8279f7685
Date: 2024-05-07T05:13:33.891Z
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Windows_NT x64 10.0.19045

**Comment by gitstua** [comment]
👋🏽  Hi @michaeltaman, sorry to hear you are having challenges.

You should consider raising a support case. In the meantime maybe double check the info on the [docs site ](https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-network-errors-for-github-copilot)around network troubleshooting. The error you are receiving may indicate proxy or firewall issues..

**Comment by orshemtov** [comment]
same here

![image](https://github.com/community/community/assets/64565195/5e44ca9f-b52f-429e-b11e-ff11b466ca29)

**Comment by yanivamrami** [comment]
Same here. happened right after the last update - 
VS Code Version: 1.89.0 (Universal)
OS: Darwin arm64 23.4.0
GitHub Copilot Chat v0.15.2
GitHub Copilot v1.194.0

**Comment by gitstua** [comment, marked as answer]
We've had some confirmation from others that this appears to be an issue with a specific ISP. Please contact your ISP for assistance here.

**Reply by yanivamrami** [reply]
That is weird.

I am clicking the "Sign In" button, VS Code opens the "The extnsion 'GitHub...' want to sign in..."
I am clicking Allow and it opens GitHub website and asks me to open VSCode.
I am clicking the 'Open Visual Studio Code'

Not sure how it is related to my ISP?!
No need to say I do have network to other locations / services...

**Reply by hsenag** [reply]
@yanivamrami Try visiting https://api.github.com in a browser, I think it'll show you a page from your ISP saying that it's been blocked.

**Reply by michaeltaman** [reply]
Hi. Via my mobile phone I able to recive responce from https://api.github.com , but from my PC I'm unable receiving such response. All other sites from my PC continues to work normally.

**Reply by yanivamrami** [reply]
Same here. mobile - works fine. Computer - cannot get to api.github.com
Where are you from? maybe it is a CDN issue or local DNS? I am based on Israel if it helps...

**Reply by hsenag** [reply]
For all the reports we've seen where we could identify the ISP, it was Bezek

**Reply by hsenag** [reply]
If you can't get to https://api.github.com/ in a browser (if working properly, it would show you some json text), then that is the problem that you should report to the ISP.

**Reply by yanivamrami** [reply]
Thanks, I'll reach out to them ASAP.

**Reply by avipars** [reply]
> Thanks, I'll reach out to them ASAP.

So bezeq is blocking it?

I have my own dns servers set up (not via the isp)... and still can't get to the ip address 
3.127.156.149

But copilot.com, github.com, api.github.com work

**Comment by michaeltaman** [comment]
Only now, when connecting through a mobile phone, and not through the home network (that is, I changed the Internet provider), I was again able to receive answers from the co-pilot's chat. So the problem was with the provider. Thank all for assistance.

**Comment by alifares10** [comment]
did anyone contact bezeq ?

**Comment by michaeltaman** [comment]
Thank you. The problem is solved now.
Regards,
Michael.

**Reply by yanivamrami** [reply]
Hi @michaeltaman how did you resovle it? what did you say to Bezeq?
It is a bit challenging for them apparently...

Thanks

**Comment by michaeltaman** [comment]
Call to me 0507158276.
